modules/running-distributed-data-science-workloads-from-notebooks.adoc

```diff
@@ -8,27 +8,77 @@ To run a distributed data science workload from a notebook, you must first provi
 See the table in link:{odhdocshome}/working_on_data_science_projects/#notebook-images-for-data-scientists_notebooks[Notebook images for data scientists] for a complete list of default notebook images.
 endif::[]
 * You have launched your notebook server and logged in to Jupyter.
 
 .Procedure
-. To access the demo notebooks, clone the `codeflare-sdk` repository as follows:
+. Download the demo notebooks provided by the CodeFlare SDK.
+The demo notebooks provide guidelines for how to use the CodeFlare stack in your own notebooks.
++
+To access the demo notebooks, clone the `codeflare-sdk` repository as follows:
+
 .. In the JupyterLab interface, click *Git > Clone a Repository*.
 .. In the "Clone a repo" dialog, enter `https://github.com/project-codeflare/codeflare-sdk.git` and then click *Clone*.
 The `codeflare-sdk` repository is listed in the left navigation pane.
 . Run a distributed workload job as shown in the following example:
 .. In the JupyterLab interface, in the left navigation pane, double-click *codeflare-sdk*.
 .. Double-click *demo-notebooks*, and then double-click *guided-demos*.
 .. Update each example demo notebook as follows:
-** Replace the links to the example community image with a link to your Ray cluster image.
+** Update the `import` section to import the `generate_cert` component:
++
+.Updated import section
+[source,bash]
+----
+from codeflare_sdk import generate_cert
+----
+
+** Replace the `default` namespace value with the name of your data science project.
+** Replace the link to the example community image with a link to your Ray cluster image.
+** Add the following Ray cluster authentication code after the Ray cluster creation step. Replace _your_cluster_name_ and _your_namespace_ with the values for your Ray cluster name and your data science project name.
++
+.Example Ray cluster authentication code
+[source,bash,subs="+quotes"]
+----
+generate_cert.generate_tls_cert(_your_cluster_name_, _your_namespace_)
+generate_cert.export_env(_your_cluster_name_, _your_namespace_)
+----
++
+[NOTE]
+====
+Mutual Transport Layer Security (mTLS) is enabled by default in the CodeFlare component in {productname-short}.
+Therefore, you must run the Ray cluster authentication code to create a secure connection between the Ray client that runs within a notebook and the Ray head client endpoint.
+====
+
+
+ifndef::upstream[]
+** If you have not configured a default local queue by including the `kueue.x-k8s.io/default-queue: 'true'` annotation as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-distributed-workloads_distributed-workloads#configuring-quota-management-for-distributed-workloads_distributed_workloads[Configuring quota management for distributed workloads], update the `ClusterConfiguration` code to specify the local queue for the Ray cluster, as shown in the following example:
```

Comment:
Just a suggestion, we can use 'parameter' word instead of 'code' for the line - 

update the `ClusterConfiguration` code to specify the local queue

Reply:
Replaced "`ClusterConfiguration` code" with "`ClusterConfiguration` section" for consistency with existing text.